resources/assets/less/forum/base.less

```diff
@@ -928,6 +928,27 @@ body.community {
   }
 }
 
+.forum-post-box-warning {
+  &:extend(.forum-post);
+  background: @dark-grey;
+  color: white;
+  min-height: 0;
+  padding: 0;
+
+  p {
+    line-height: 22px;
```

Comment:
a `display: flex` on the parent and `align-self: center` here works better, I believe (and guarantees centering).